shaptools/hdb_connector/connectors/dbapi_connector.py

```diff
@@ -59,10 +59,11 @@ def query(self, sql_statement):
             with self._connection.cursor() as cursor:
                 cursor.execute(sql_statement)
                 result = cursor.fetchall()
+                meta_data = cursor.description
         except dbapi.Error as err:
             raise base_connector.QueryError('query failed: {}'.format(err))
         self._logger.info('query result: %s' % result)
```

Comment:
Can we have the same log trace in both connectors? (and udpate the docstring too)